datadog-agent-nvml.yaml

```diff
@@ -0,0 +1,132 @@
+package:
+  name: datadog-agent-nvml
+  # This package has two git checkouts. For each new release, the commit SHA for
+  # DataDog/integrations-core must also be updated.
+  version: 1.0.9
+  epoch: 0
+  description: "Checks NVIDIA Management Library (NVML) exposed metrics through the Datadog Agent and can correlate them with the exposed Kubernetes devices"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - datadog-agent # always pick the latest agent to run the `integration install` c ommand.
+      - datadog-agent-core-integrations
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - datadog-agent
+      - datadog-agent-core-integrations
+      - rsync
+
+vars:
+  DD_HOME: / # agent being run by root expects /.
```

Comment:
The `agent integration` commands for some reason resolves the home path to `/` instead of `/opt/datadog-agent/` as it does in the upstream container image.
It needs to be investigated and fixed in the main `datadog-agent` probably.